_kbarticles/2024-12-10-using-opensearch-dashboards.md

```diff
@@ -0,0 +1,71 @@
+m ---
```

Comment:
```suggestion
---
```

I wonder if this stray character is messing up the front matter and redirect

Reply:
Yes, that was it. That typo snuck in between commits, really confusing me....